### PR TITLE
[Project Submission] Creative

### PR DESCRIPTION
**Pull request by gawainb**
Per comment: https://github.com/filecoin-project/community/pull/328#issuecomment-947283485

### CODE REVIEW

**Comment by longfeiWan9** [comment]
@gawainb Thank you for updating it. Since you update it in a separate PR, could you please update the title of this PR to `[Project Submission] Creative` and close [PR-328](https://github.com/filecoin-project/community/pull/328)

**Comment by gawainb** [comment]
> @gawainb Thank you for updating it. Since you update it in a separate PR, could you please update the title of this PR to `[Project Submission] Creative` and close [PR-328](https://github.com/filecoin-project/community/pull/328)

Thank you, completed comment.

File: projects/application/Creative.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Creative
+<!-- Add you project logo here if available-->
+![Creative Logo](https://avatars.githubusercontent.com/u/76276990?s=400&u=5f00ea3b39eb69256005d0c293ffbdcc4de69cc4&v=4)
+
+## Category 
+<!--developer tooling, application, wallet, infrastructure, etc-->
+Application
+
+## Project Description
+<!--Describe your project in a few sentences. -->
```
Comment (longfeiWan9) [inline projects/application/Creative.md:12]: In filecoin-project/devgrants#346, you have mentioned that this project is using Textile to store content on Filecoin, and also awesome future plan to integrate Filecoin storage into our existing Textile implementation using the Polygon/Filecoin bridge. One extra thing I noticed that you hosted the website with Fleek on IPFS/Filecoin which is awesome. 

I think those information is very interesting and important for our community as well. So could you please add something like that in the description section to let everyone know more about it?

File: projects/application/Creative.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+<!--Describe who will be your project's users-->
+Creators, brands and fans
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+N/A
+
+## Github repo
+<!--Attach a link to your GitHub repo if it's OSS-->
+UI: https://github.com/g2entgroup/creative-ui-2.2
```
Comment (longfeiWan9) [inline projects/application/Creative.md:36]:
```suggestion
+ UI: https://github.com/g2entgroup/creative-ui-2.2
```

File: projects/application/Creative.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Contracts: https://github.com/g2entgroup/creative-contracts
```
Comment (longfeiWan9) [inline projects/application/Creative.md:37]:
```suggestion
+ Contracts: https://github.com/g2entgroup/creative-contracts
```

File: projects/application/Creative.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+### Team Size
+3
+
+### Team members  
+Gawain Bracy II - Founder, [Github](https://github.com/gawainb)
+Torch Ablaze - Frontend Developer, [Github](https://github.com/Rashmi-278)
+Anmol Agarwala - Backend Developer, [Github](https://github.com/sylar217)
```
Comment (longfeiWan9) [inline projects/application/Creative.md:55]:
```suggestion
+ Gawain Bracy II - Founder, [Github](https://github.com/gawainb)
+ Torch Ablaze - Frontend Developer, [Github](https://github.com/Rashmi-278)
+ Anmol Agarwala - Backend Developer, [Github](https://github.com/sylar217)
```

File: projects/application/Creative.md (hunk 3 above)
Comment (longfeiWan9) [inline projects/application/Creative.md:37]:
```suggestion
+ UI: https://github.com/g2entgroup/creative-ui-2.2
+ Contracts: https://github.com/g2entgroup/creative-contracts
```

File: projects/application/Creative.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+## How the community can engage
+GitHub Discussion: GitHub Discussion: https://github.com/filecoin-project/community/discussions/327   
+Email: cryptocreativeorg@gmail.com   
+Twitter:  https://twitter.com/creativecrtv
+Discord:  https://discord.gg/rGNUvmxe
+Telegram:  https://t.me/joinchat/RQ-p_QD7XIA2gDYO
```
Comment (longfeiWan9) [inline projects/application/Creative.md:62]:
```suggestion
+ GitHub Discussion: https://github.com/filecoin-project/community/discussions/327   
+ Email: cryptocreativeorg@gmail.com   
+ Twitter:  https://twitter.com/creativecrtv
+ Discord:  https://discord.gg/rGNUvmxe
+ Telegram:  https://t.me/joinchat/RQ-p_QD7XIA2gDYO  
```

**Review by longfeiWan9 — APPROVED** [review]

**Pull request merged**